include/chainbase/environment.hpp

```diff
@@ -5,7 +5,7 @@
 namespace chainbase {
 
 constexpr size_t header_size = 1024;
-constexpr uint64_t header_id = 0x3242444f49534f45ULL; //"EOSIODB2" little endian
+constexpr uint64_t header_id = 0x3342444f49534f45ULL; //"EOSIODB3" little endian
```

Comment:
What does the error message in leap look like with this?

Reply:
:+1: if we we don't already have an existing discrete exception type that bubbles up eventually to a nice error message on nodeos, we should consider that

Reply:
haha, not quite user-friendly (see below). It looks like we didn't even need my change 
But please approve this `chainbase` PR, and then I can try impriving the error messahe in the `Leap` PR.

![image](https://github.com/AntelopeIO/chainbase/assets/1496907/105c9362-572c-4688-826b-910b9ec78c03)